app/generators/description.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-module CocinaGenerators
+module Generators
```

Comment:
`CocinaGenerators` is clearer to me then `Generators` since they generate Cocina.

Reply:
Totally fine by me.